developer-docs/dev_setup.md

```diff
@@ -1,215 +1,145 @@
-Server Version: version.Info{Major:"1", Minor:"4", GitVersion:"v1.4.5", GitCommit:"5a0a696437ad35c133c0c8493f7e9d22b0f9b81b", GitTreeState:"clean", BuildDate:"1970-01-01T00:00:00Z", GoVersion:"go1.7.1", Compiler:"gc", Platform:"linux/amd64"}
-```
-
-### Configure kubectl
-
-You will need to configure the `kubectl` client with the appropriate details to be able to talk to the kubernetes cluster.
-We recommend copying the [example kubectl config](resources/kubeconfig) to `~/.kube/config`. Note that you may need to create this directory and file.
-
-The only change you will need to make is to replace _"XXXXXXXXXX"_ with your kubernetes token.
-
-You can submit a request to obtain a kubernetes token by adding an issue to the Hosting Platform Bau project on Github [here](https://github.com/UKHomeOffice/application-container-platform-bau/issues).
-
-Please use the below template for your request:
-
-```
-Please can I have a kubernetes token with access to my teams namespaces.  
-Email: xxx.xxx@digital.homeoffice.gov.uk  
-Name: xxx xxx  
-Team: My Project Team  
-Public GPG Key: xxxxxxxxx
-Namespace: dev-induction
+$ openvpn --version
+OpenVPN 2.3.13 x86_64-apple-darwin16.1.0 [SSL (OpenSSL)] [LZO] [MH] [IPv6] built on Nov  5 2016
 ```
 
-You need to provide your public gpg key as the kube token you receive back will be encrypted using it.
-If you need to, you can [generate a gpg key](https://help.github.com/articles/generating-a-new-gpg-key/).
-
-When you've received back the token, you will need to decrypt it and use the resulting token string
-as your token in your `~/.kube/config` file.
+#### Install Docker
 
-To do this run the following command with the file you received back.
+You can follow the instructions to install `docker` from the [Docker website]. You can verify the installation is successful with:
 
 ```bash
-$ gpg -d <your encrypted_token.gpg>
+$ docker --version
+Docker version 17.12.0-ce, build c97c6d6
 ```
 
-Then enter the passphrase you entered when creating the initial gpg key that was sent in the issue to the Hosting Platform Bau project.
+#### Install kubectl
 
-Following this you should have some information printed to the console that follows this example:
+You can follow the instructions to install `kubectl` from the [Kubernetes website]. You can verify the installation is successful with:
 
 ```bash
-You need a passphrase to unlock the secret key for
-user: "John Smith <john.smith@xxxxxxx.xxxxxxxxxx.xxx.xx>"
-4096-bit RSA key, ID AAAAAAAA, created 2000-01-01 (main key ID BBBBBBBB)
-
-gpg: encrypted with 4096-bit RSA key, ID BBBBBBBB, created 2000-01-01
-      "John Smith <john.smith@xxxxxxx.xxxxxxxxxx.xxx.xx>"
-XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX
+$ kubectl version
+Client Version: version.Info{Major:"1", Minor:"7", GitVersion:"v1.7.3", GitCommit:"2c2fe6e8278a5db2d15a013987b53968c743f2a1", GitTreeState:"clean", BuildDate:"2017-08-03T07:00:21Z", GoVersion:"go1.8.3", Compiler:"gc", Platform:"linux/amd64"}
```

Comment:
Put this and KD last and rather than this test, please ask them to list secrets and pods in developer-induction ns on acp-notprod. That would be a much better way of checking all is well.